### DISCUSSION TITLE
How to make mobile mode use 2 columns for thumbnail list?

### DISCUSSION THREAD

**Discussion by ramabondanp** [Difegue/LANraragi#502] [Q&A] [answered]
Just want to ask if there is a way to make mobile mode use 2 columns for thumbnail list?

**Comment by Difegue** [comment]
There's no special handling for mobile/small screens atm so no, although you could probably make it work by modifying your device's default DPI?  (Admittedly kind of a clunky approach)
https://beebom.com/how-change-dpi-android

It might be good to make the thumbnails a bit smaller for phone screens in the future so that we could fit two of them.

**Comment by Difegue** [comment, marked as answer]
I've made some improvements for small screens in https://github.com/Difegue/LANraragi/commit/07780b304a91ca65a9705391c1e96e50c333db1f .
